extensions/amp-form/0.1/amp-form.js

```diff
@@ -431,6 +432,15 @@ export class AmpForm {
       checkUserValidityAfterInteraction_(dev().assertElement(e.target));
       this.validator_.onInput(e);
     });
+
+    // Ctrl/Cmd + Enter on textarea or contenteditable triggers form submission.
+    this.form_.querySelectorAll('textarea,[contenteditable]').forEach(el =>
+      el.addEventListener('keydown', e => {
+        if (e.key == Keys.ENTER && (e.ctrlKey || e.metaKey)) {
```

Comment:
This will trigger on Windows Key + Enter too btw but maybe that's expected?

Reply:
Sometimes -- the Windows key as `metaKey` appears to be [inconsistently supported across browsers](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey).

Reply:
Sg